src/server.c

```diff
@@ -555,14 +555,16 @@ hashtableType setHashtableType = {
     .keyCompare = hashtableSdsKeyCompare,
     .entryDestructor = dictSdsDestructor};
 
+const void *zsetHashtableGetKey(const void *element) {
+    zskiplistNode *node = (zskiplistNode *)element;
```

Comment:
Implicit cast should work if we keep `const`?

```suggestion
    const zskiplistNode *node = element;
```